src/components/EditPasswordForm/inplayer/EditPasswordForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```

Comment:
I would suggest we rename the current EditPasswordForm.tsx to ResetPasswordForm.tsx and create this one in its place without the inplayer subdirectory.

Reply:
hmm... I am not sure. we have different flows. Cleeng resets the password via email.
InPlayer has 2 ways of changing the password:

1. With an old password
2. With reset token ( forgot password flow )

https://www.youtube.com/watch?v=UNcvHDsQJV4

Reply:
That's fine. Those are indeed 3 distinct flows, which should have 3 distinct UI's. I think there can be 3 components ResetPasswordByEmailForm, ResetPasswordByTokenForm, and EditPasswordForm. There doesn't need to be any grouping or cleeng/inplayer nomenclature at the component level. These 3 password manipulation types could be used by other different auth providers in future code. If at all possible, I think it's best to try to avoid UI  components coupled to an implementation.

Reply:
I guess the edit password and reset by token flows are similar enough there's no reason to split them. I think `EditPasswordForm` and `ResetPasswordForm` or `ResetPasswordRequestForm` or something are discernible enough names to differentiate the 2.

Reply:
yes, this make sense